wandb/sdk/launch/builder/build.py

```diff
@@ -572,3 +540,19 @@ def image_tag_from_dockerfile_and_source(
     unique_id_string = image_source_string + dockerfile_contents
     image_tag = hashlib.sha256(unique_id_string.encode("utf-8")).hexdigest()[:8]
     return image_tag
+
+
+def _inject_wandb_config_env_vars(
+    config: Dict[str, Any], env_dict: Dict[str, Any], maximum_env_length: int
+) -> None:
+    str_config = json.dumps(config)
+    if len(str_config) <= maximum_env_length:
+        env_dict["WANDB_CONFIG"] = str_config
```

Comment:
if we make this `WANDB_CONFIG_0` then we don't need special handling for chunked vs. normal case

Reply:
we would have to slightly change your handling below to account for older versions, so maybe not worth

Reply:
The purpose of maintaining support for the original WANDB_CONFIG is when users use a newer launch agent to run older versions of wandb